test/RedisSessionStateProviderFunctionalTests/RedisSessionStateProvider.FunctionalTests.csproj

```diff
@@ -16,19 +16,19 @@
     <ProjectReference Include="..\..\src\RedisSessionStateProvider\RedisSessionStateProvider.csproj" />
```

Comment:
We should add a functional test that provides its own `ISessionDataSerializer` to test that scenario, and demonstrate how others should do it

Reply:
Added